exporters/otlp/README.md

```diff
@@ -16,23 +16,19 @@ $ go get -u go.opentelemetry.io/otel/exporters/otlp
 A new exporter can be created using the `NewExporter` function.
 
 ```golang
```

Comment:
Can we remove this code block entirely so we don't have any other stale code in the future?